packages/forms-web-app/src/controllers/appellant-submission/confirmation.js

```diff
@@ -1,12 +1,18 @@
 const { VIEW } = require('../../lib/views');
+const { isDecisionDatePassed } = require('../eligibility/decision-date');
 
 exports.getConfirmation = (req, res) => {
   const appellantEmail = req.session && req.session.appeal && req.session.appeal['appellant-email'];
   const appealId = req.session.appeal.id;
-  req.session.appeal = null;
+  const decisionDatePassed = isDecisionDatePassed(req.session.appeal);
 
-  res.render(VIEW.APPELLANT_SUBMISSION.CONFIRMATION, {
-    appellantEmail,
-    appealId,
-  });
+  if (decisionDatePassed) {
+    res.redirect(`/${VIEW.ELIGIBILITY.DECISION_DATE_PASSED}`);
+  } else {
+    req.session.appeal = null;
```

Comment:
Is this simply to clean up? 

Also, I can't see a test for the path where it becomes nullified.